lib/context.js

```diff
@@ -108,7 +108,7 @@ const proto = module.exports = {
     // don't do anything if there is no error.
     // this allows you to pass `this.onerror`
     // to node-style callbacks.
-    if (null == err) return;
```

Comment:
`exp: null == undefined -> true`. This is what we expect here. :(

Reply:
How  to capture the `non-error` error when passing `undefined`, cause you have already  returned.